# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../frozen-bubble/create-server-dependancies.patch
# Copyright (C) 2007 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

diff -ru frozen-bubble-2.1.0.org/server/Makefile frozen-bubble-2.1.0/server/Makefile
--- frozen-bubble-2.1.0.org/server/Makefile	2006-11-12 20:56:02.000000000 +0100
+++ frozen-bubble-2.1.0/server/Makefile	2007-04-15 14:05:40.000000000 +0200
@@ -25,7 +25,10 @@
 	install fb-server $(DESTDIR)$(LIBDIR)/frozen-bubble
 
 .depend:
-	$(CPP) $(CFLAGS) $(CPPFLAGS) -M $(ALLSRC) > .depend
+	rm -f .depend
+	for src in $(ALLSRC); do \
+		$(CPP) $(CFLAGS) $(CPPFLAGS) -M $$src >> .depend; \
+	done
 
 ifeq (.depend,$(wildcard .depend))
 include .depend
